std/internal/math/gammafunction.d

```diff
@@ -231,7 +238,12 @@ real igammaTemmeLarge(real a, real x)
 
 public:
 /// The maximum value of x for which gamma(x) < real.infinity.
-enum real MAXGAMMA = 1755.5483429L;
+static if (real.mant_dig == 64)  // 80-bit reals
```

Comment:
I think now I screwed up this branch/PR while trying to update it to upstream master :'(